services/traefik/34.1.0/crds.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: traefik-crd-helmrelease
+  namespace: ${releaseNamespace}
+spec:
+  dependsOn:
+    # There can only be one operator per cluster, just ensure CRDs are present, no need to wait for actual operator Deployment.
+    - name: traefik-crd-prereq-jobs-v34.1.0
+      namespace: ${workspaceNamespace}
```

Comment:
This Kustomization is installing the CRD, so there is no need to add `dependsOn`. However the `traefik` HR installation should be moved to a separate kustomization and should wait